### DISCUSSION TITLE
CDS Installation

### DISCUSSION THREAD

**Discussion by msadikkose** [indrawp/Setup-Guide#49] [Ideas]
Hello @madeindra 
Thanks for such a detailed documentation. 
Would you consider adding CDS installation steps too. Because I could not find an updated documentation and it seems a bit challanging. Or is there any online CDS that can be integrated signal server. 
Regards.

**Comment by madeindra** [comment]
@msadikkose I will once I got a device with Intel SGX

**Reply by msadikkose** [reply]
Hello @madeindra , 
Maybe you can try a vm from Azure, AWS or Google just for tutorial. So while waiting for a CDS installation documentation from your side, how can I test v2.92 without CDS ? Should I just remove CDS configurations from yml file ? 
Thanks

**Reply by madeindra** [reply]
@msadikkose It's a bit pricey for me, SGX enabled VM is not cheap by the way.
For v2.92 it can work finely without cds, just follow the guide, no other steps left from the guide
